jax/numpy/lax_numpy.py

```diff
@@ -1537,8 +1537,13 @@ def _isposneginf(infinity, x):
   else:
     return full_like(x, False, dtype=bool_)
 
-isposinf = _wraps(np.isposinf)(partial(_isposneginf, inf))
-isneginf = _wraps(np.isneginf)(partial(_isposneginf, -inf))
+@_wraps(np.isposinf)
+def isposinf(x):
```

Comment:
I'm going to channel my inner mattjj@ here and say: could we write this:
```
isposinf = _wraps(np.isposinf)(lambda x: _isposneginf(inf, x))
```
to save several lines?

Reply:
Done.